src/sdg/api/serializers/core.py

```diff
@@ -29,9 +34,27 @@ class Meta:
             "url": {
                 "view_name": "api:productencatalogus-detail",
                 "lookup_field": "uuid",
+                "help_text": "De url van de specifieke catalogi api call",
+            },
+            "uuid": {
+                "help_text": "De UUID(https://en.wikipedia.org/wiki/Universally_unique_identifier) van de specifieke catalogi"
+            },
+            "domein": {
+                "help_text": "Een afkorting die wordt gebruikt om het domein aan te duiden."
+            },
+            "naam": {"help_text": "De naam van de catalogi"},
+            "is_referentie_catalogus": {
+                "help_text": "Dit is een boolean die aangeeft of de catalogus een referentie catalogus is, als deze op false staat dan is het een producten catalogus."
             },
             "referentie_catalogus": {
                 "lookup_field": "uuid",
                 "view_name": "api:productencatalogus-detail",
+                "help_text": "Dit is de api call url van de referentie catalogus waar dit product aan gekoppeld is.",
```

Comment:
Volgens mij bedoel je hier "de URL van de referentieproductcatalogus waar deze productcatalogus aan gekoppeld is", toch?